nix/shell.nix

```diff
@@ -5,6 +5,7 @@ pkgs.mkShell {
     rustToolchain
     rust-analyzer
 
+    nodejs_20
```

Comment:
Is this necessary?

Reply:
ups, this shouldnt got pushed. Enabled gh copilot on my side.